### PR TITLE
readme: add some details and usage

### PR DESCRIPTION
**Pull request by abravalheri**
Thank you very much for the handy library @FFY00.

I tried to put up together a PR to add some clarifications and usage to the README.
Hopefully it is useful for you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,32 @@
...
+
+After [installing `pep621`](https://pypi.org/project/pep621/),
+you can use it as a library in your scripts and programs:
+
+```python
+from pep621 import StandardMetadata
+
+parsed_pyproject = { ... }  # you can use parsers like `tomli` to obtain this dict
+metadata = StandardMetadata.from_pyproject(parsed_pyproject)
+print(metadata.entrypoints)  # Same fields as defined in PEP 621
```
Comment (FFY00) [inline README.md:30]:
```suggestion
print(metadata.entrypoints)  # same fields as defined in PEP 621
```

File: README.md [hunk 2]
```diff
@@ -6,4 +6,32 @@
 [![codecov](https://codecov.io/gh/FFY00/python-pep621/branch/main/graph/badge.svg?token=9chBjS1lch)](https://codecov.io/gh/FFY00/python-pep621)
 [![Documentation Status](https://readthedocs.org/projects/pep621/badge/?version=latest)](https://pep621.readthedocs.io/en/latest/?badge=latest)
 
-PEP 621 metadata parsing.
+
+> Data class for PEP 621 metadata with support for [core metadata] generation
```
Comment (FFY00) [inline README.md:10]:
```suggestion
> Dataclass for PEP 621 metadata with support for [core metadata] generation
```

**Review by FFY00 — APPROVED** [review]
Thank you! There are some minor nits.

**Comment by abravalheri** [comment]
Thank you very much for the review @FFY00

**Pull request merged**